easyInterface/Diffraction/Interface.py

```diff
@@ -450,7 +450,12 @@ def getCalculation(self, calculation_name: str) -> Calculation:
         return calculation
 
     def setPhase(self, phase: Union[Phase, dict]):
-        """Set phases (sample model tab in GUI)"""
+        """
+        Modify a phase in the calculator. THe phase will be added if it does not currently exist.
```

Comment:
"THe" -> "The"

Reply:
Fixed in 6c795c8955331c495428a029460a00a593f6633b